staker-rewards/rewards.js

```diff
@@ -0,0 +1,128 @@
+import clc from "cli-color"
+
+import Context from "./lib/context.js"
+import SLACalculator from "./lib/sla-calculator.js"
+
+async function run() {
+    try {
+        // URL to the websocket endpoint of the Ethereum node.
+        const ethHostname = process.env.ETH_HOSTNAME
+        // Start of the interval passed as UNIX timestamp.
+        const intervalStart = process.argv[2]
+        // End of the interval passed as UNIX timestamp.
+        const intervalEnd = process.argv[3]
+        // Determines whether debug logs should be disabled.
+        const isDebugDisabled = process.env.DEBUG !== "on"
+        // Determines whether the cache refresh process should be performed.
+        const isCacheRefreshEnabled = process.env.CACHE_REFRESH !== "off"
+
+        if (!ethHostname) {
+            console.error(clc.red("Wrong ETH_HOSTNAME value"))
+            return
+        }
+
+        const interval = {
+            start: intervalStart,
+            end: intervalEnd
+        }
+
+        validateIntervalTimestamps(interval)
+
+        if (isDebugDisabled) {
+            console.debug = function() {}
+        }
+
+        const context = await Context.initialize(ethHostname)
+
+        const { cache } = context
+
+        if (isCacheRefreshEnabled) {
+            console.log("Refreshing keeps cache...")
+            await cache.refresh()
+        }
+
+        const operatorsRewards = calculateOperatorsRewards(cache, interval)
+
+        console.table(operatorsRewards)
+    } catch (error) {
+        throw new Error(error)
+    }
+}
+
+function validateIntervalTimestamps(interval) {
+    const startDate = new Date(interval.start * 1000)
+    const endDate = new Date(interval.end * 1000)
+
+    const isValidStartDate = startDate.getTime() > 0
+    if (!isValidStartDate) {
+        throw new Error("Invalid interval start timestamp")
+    }
+
+    const isValidEndDate = endDate.getTime() > 0
+    if (!isValidEndDate) {
+        throw new Error("Invalid interval end timestamp")
+    }
+
+    const isEndAfterStart = endDate.getTime() > startDate.getTime()
+    if (!isEndAfterStart) {
+        throw new Error(
+            "Interval end timestamp should be bigger than the interval start"
+        )
+    }
+
+    console.log(clc.green(`Interval start: ${startDate.toISOString()}`))
+    console.log(clc.green(`Interval end: ${endDate.toISOString()}`))
+}
+
+function calculateOperatorsRewards(cache, interval) {
+    const slaCalculator = SLACalculator.initialize(cache, interval)
+
+    const operatorsRewards = []
+
+    for (const operator of getOperators(cache)) {
+        const operatorSLA = slaCalculator.calculateOperatorSLA(operator)
+        // TODO: other computations
+
+        operatorsRewards.push(
+            new OperatorRewards(
+                operator,
+                operatorSLA
+            )
+        )
```

Comment:
We can output here all operators but let's keep in mind we should merkelize only non-zero reward transfers (cc @nkuba, @dimpar).